### DISCUSSION TITLE
auth0_guardian resource has risk assessment field separate to policy field

### DISCUSSION THREAD

**Discussion by Willrholmes** [auth0/terraform-provider-auth0#1178] [General]
### Checklist

- [X] I agree to the terms within the [Auth0 Code of Conduct](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md).

### Describe the problem you'd like to have solved

It would be great to be able to control the risk assessment toggle separate to the the policy field for the auth0_guardian resource. Risk assessment is toggled on automatically when policy is set to `confidence-score` but cannot be turned on otherwise via this resource. Also, if you change the value of the policy field from `confidence-score`  back to `never` the confidence score remains toggled on.

The dashboard allows this separate control already so it must be possible via the API.

<img width="950" alt="image" src="https://github.com/auth0/terraform-provider-auth0/assets/30688262/561e395f-2bdd-4ff8-bc35-8afb1bec34c9">

### Describe the ideal solution

Add a new field that controls risk assessment (true/false) to the auth0_guardian resource.

**Comment by sergiught** [comment]
Hey @Willrholmes 👋🏻 

Thanks for taking the time to raise this with us. We'll look into your request and circle back once we have an update.

**Comment by dmoughabghab** [comment]
+One here, can confirm what is being said and it is useful sometimes to have on separately to the confidence score

**Comment by bevel-zgates** [comment]
We just ran into this. We've shimmed it now by implementing our own PATCH to 
https://manage.auth0.com/api/risk-assessment/config
with { AfterAuthentications : bool }

But it'd be nice to have this supported in the tf provider.
